SwitchItem/SwitchItem.module.less

```diff
@@ -6,6 +6,13 @@
 .switchItem {
 	.label {
 		vertical-align: middle;
+		position: absolute;
+		right: 0;
+
+		.enact-locale-rtl({
+			right: initial;
+			left: 0;
+		});
 	}
```

Comment:
I think this class can be removed if we could make the before and after label in Item vertically center well.

Reply:
Yea, it seems this is not needed anymore because of the `Layout` alignment we did in Item.js.